tests/browsing_context/test_traverse_history.py

```diff
@@ -21,28 +21,32 @@
 
 
 @pytest.mark.asyncio
-async def test_traverse_history(websocket, context_id, html):
+async def test_traverse_history(websocket, context_id):
+    urls = []
     for i in range(HISTORY_LENGTH + 1):
-        await goto_url(websocket, context_id, html(i))
+        # TODO: use `html` fixture instead.
```

Comment:
Will you address it now or in another PR?

Reply:
I'll file an issue